### ISSUE TITLE
Clause 10: which is better example, LicenseRef-Whiskeyware or LicenseRef-Beerware-4.2

### ISSUE THREAD

**Issue by fu7mu4** [spdx/spdx-spec#605]
In 10.1.3 Examples of current SPDX, the EXAMPLE LicenseID: is  LicenseID: `LicenseRef-Beerware-4.2`
But 10.2.3 Examples the EXAMPLE 2 RDF is  `LicenseRef-Whiskeyware` :

```xml
<ExtractedLicensingInfo rdf:about="licenseRef-Whiskeyware">
    <licenseId>LicenseRef-Whiskeyware</licenseId>
    <extractedText>This software is licensed under the WHISKEY-WARE LICENSE.</extractedText>
</ExtractedLicensingInfo>
```

I think `LicenseRef-Whiskeyware` is better than `LicenseRef-Beerware-4.2` for the example LicenseID in 10.1.3 because the `Beerware License` has already its identifier , see https://spdx.org/licenses/Beerware.html

see https://spdx.github.io/spdx-spec/other-licensing-information-detected/

**Comment by seabass-labrax** [comment]
Good point, @fu7mu4! I'd agree that it makes sense to choose a license that isn't in the License List when demonstrating the use of `LicenseRef-`.
